lemminx-maven/pom.xml

```diff
@@ -89,6 +89,16 @@
 			<artifactId>wagon-http</artifactId>
 			<version>3.3.4</version>
 		</dependency>
+	<dependency>

```

Comment:
@mickaelistria these aren't required for the fix, however I found them in the m2e runtime. I can remove them realistically if they're not providing any improvements.